components/applications-service/pkg/storage/postgres/schema/sql/12_artificial_ids_for_service_groups.up.sql

```diff
@@ -0,0 +1,3 @@
+ALTER TABLE service_full ADD COLUMN service_group_id text;
```

Comment:
To explain what's going on here a bit:

When we switch to the denormalized table, we no longer have artificial integer keys for service group rows since we don't have a service group table. This is a problem because we expose the service group IDs to the UI, which uses them as IDs for the rows in the data layer, and also puts them in requests back to the API when it fetches the services for the selected service group. So we want to re-introduce the concept of an artificial service group ID which serves the same purposes. I put some questions for discussion in the `hab_event.go` diff.